app/scripts/views/marketing_snippet_ios.js

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Handles the marketing snippet on the 'ready' page.
+ *
+ * Shows `Get Sync on Firefox for Android` for users that complete
+ * signup for sync in Firefox Desktop.
+ */
+
+'use strict';
+
+define([
+  'views/base',
+  'lib/constants',
+  'stache!templates/marketing_snippet_ios'
+], function (BaseView, Constants, Template) {
+
+  var View = BaseView.extend({
+    template: Template,
+    appStoreImageLanguages: [
+      'da',
+      'de',
+      'en',
+      'es',
+      'et',
+      'fr',
+      'he',
+      'hu',
+      'id',
+      'it',
+      'ja',
+      'ko',
+      'lt',
+      'nb-NO',
+      'nl',
+      'pl',
+      'pt-BR',
+      'pt',
+      'ru',
+      'sk',
+      'sl',
+      'sv-SE',
+      'tr',
+      'zh-CN',
+      'zh-TW'
+    ],
+    playStoreImageLanguages: [
+      'ca',
+      'cs',
+      'da',
+      'de',
+      'en',
+      'es',
+      'et',
+      'fr',
+      'hu',
+      'id',
+      'it',
+      'ja',
+      'ko',
+      'lt',
+      'nb-NO',
+      'nl',
+      'pl',
+      'pt-BR',
+      'pt',
+      'ru',
+      'sk',
+      'sl',
+      'sv',
+      'tr',
+      'uk',
+      'zh-CN',
+      'zh-TW'
+    ],
+
+    initialize: function (options) {
+      options = options || {};
+
+      this._type = options.type;
+      this._service = options.service;
+      this._language = options.language;
+      this._able = options.able;
+    },
+
+    context: function () {
+      var shouldShowMarketing = this._shouldShowSignUpMarketing();
+      var iosUser = this._isIos();
+      var androidUser = this._isAndroid();
+      // app store links come from experiments
+      // allows us to update after prod push
+      var appStoreLinkDirect = this._able.choose('fxIos').appStoreLinkDirect;
```

Comment:
I was thinking about this approach last night. I think this will break if the experiment `fxIos` has not yet passed the startDate. `choose` will return `undefined`, which will cause the `.appStoreLinkDirect` reference to explode.